### PR TITLE
Early pull request with Delete functionality. 

### PR DESCRIPTION
**Pull request by thesmallstar**

### CODE REVIEW

**Comment by Layman806** [comment]
You don't want to have 'dd' as a command inside there. Conflicts with 'dd' - 'Data Destroyer' the linux tool.
Instead of using 'dd' or 'df', you could have a single command: 'del <name>'. Then stat for the file/dir and then delete it.

**Comment by thesmallstar** [comment]
@Layman806  Thank you for pointing it out!

**Comment by thesmallstar** [comment]
Although I think it should be perfectly fine having dd as one of the commands since it is taken as an input by the program, it should not interpret it as the Linux command as we are using it only for reference purpose. 
For the safe side, I will change it.

**Comment by debck** [comment]
The changes you made in the `app.js` (adding more else if statements ) should be done in `sub.js` . The `app.js` file should contain only [this](https://github.com/debck/File-Creator-CLI/blob/master/bin/app.js)

File: lib/create.js [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-	return 0;
-};
 
-exports.createdir = (dirname) => {
-	const path = `${process.cwd()}/${dirname}`;
-	shell.mkdir(path);
+exports.create = filename => {
+  const filePath = `${process.cwd()}/${filename}`;
+  // console.log(filePath);
+  shell.touch(filePath);
```
Comment (debck) [inline lib/create.js]: you should `return 0` after line 8 so that the test passes

File: lib/create.js [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
+};
 
-	return 0;
-}
+exports.createdir = dirname => {
+  const path = `${process.cwd()}/${dirname}`;
+  shell.mkdir(path);
```
Comment (debck) [inline lib/create.js]: you should `return 0`  after line 13 so that the test passes

**Comment by thesmallstar** [comment]
@debck  please review again :)

**Comment by debck** [comment]
LGTM 👍 
Just delete the file `bb.txt ` and `r.txt` inside `bin` folder and squash your commits

**Comment by thesmallstar** [comment]
Oh sorry for that! That was part of my tests :P

**Comment by thesmallstar** [comment]
I am facing problems in squashing the commits. I will do it asap.

**Comment by debck** [comment]
@thesmallstar  you can take some help from [here](https://stackoverflow.com/questions/5667884/how-to-squash-commits-in-git-after-they-have-been-pushed/) and if you are facing some issues, you can ask in the slack

**Comment by debck** [comment]
U didn't made the squash from `master` branch.  Its fine, it takes time to learn git commands.Follow these steps to squash 
```
git checkout master
git reset --soft HEAD~(no of commits to squash)
git commmit -m"(Your message)"
git push -f origin master
```

**Comment by thesmallstar** [comment]
@debck  Thank you :) It worked.

**Comment by debck** [comment]
@thesmallstar  I will open a new issue to update the readme `usage` part. As you have worked on some imp functionality, you can easily write it. 
Tell me if you want to take the issue 👍

**Comment by thesmallstar** [comment]
@debck  Sorry I did not see your earlier message. Yes, I would like to work on the read me part :)

**Comment by debck** [comment]
@thesmallstar sorry but now someone else is working on that .
You can work on something else

**Pull request merged**